plugins/csv-adapter/src/main/java/org/polypheny/db/adapter/csv/CsvSchema.java

```diff
@@ -112,7 +121,14 @@ public CsvTable createCsvTable( long id, PhysicalTable table, CsvSource csvSourc
 
 
     /**
-     * Creates different subtype of table based on the "flavor" attribute.
+     * Creates different subtype of table based on thString tableName = entry.getKey();
+     * if ( catalog.getSnapshot().rel().getTable( namespace, tableName ).isPresent() ) {
+     * int i = 0;
+     * while ( catalog.getSnapshot().rel().getTable( namespace, tableName + i ).isPresent() ) {
+     * i++;
+     * }
+     * tableName += i;
+     * }e "flavor" attribute.
```

Comment:
Looks like a copy/paste error.

Reply:
This was already there before my changes. It's removed now.